components/compositing_traits/lib.rs

```diff
@@ -16,6 +21,8 @@ use msg::constellation_msg::{PipelineId, TopLevelBrowsingContextId};
 use net_traits::image::base::Image;
 use profile_traits::mem;
 use profile_traits::time;
+use script_traits::ConstellationControlMsg;
+use script_traits::LayoutControlMsg;
 use script_traits::{AnimationState, EventResult, MouseButton, MouseEventType};
```

Comment:
Since this file is new, these could all be collected on the the same line as well. We are gradually moving to this style, I think. Eventually we should probably configure rustfmt to enforce it.

Reply:
done in 96eec79caf5fb6dbfab8adaea382b78ca2bb8076